src/rwe/GameScene.cpp

```diff
@@ -217,7 +218,7 @@ namespace rwe
             auto rectWidth = (rect.width * std::max(Energy(0), localPlayer.energy).value) / localPlayer.maxEnergy.value;
             const auto& colorIndex = localSideData.energyColor;
             const auto& color = sceneContext.palette->at(colorIndex);
-            chromeUiRenderService.fillColor(rect.x, rect.y, rectWidth, rect.height, color);
+            chromeUiRenderService.fillColor(static_cast<float>(rect.x), static_cast<float>(rect.y), rectWidth, static_cast<float>(rect.height), color);
```

Comment:
should probably have a `fillColor` overload that takes ints instead of casting here